packages/loader/container-loader/src/container.ts

```diff
@@ -163,6 +168,12 @@ const savedContainerEvent = "saved";
 
 const packageNotFactoryError = "Code package does not implement IRuntimeFactory";
 
+/**
+ * The current generation of the Loader. This is used to determine compatibility between other layers.
```

Comment:
`The current generation of the Loader. This is used to determine it's compatibility with other layers.`

Reply:
Consider adding recommendation in notes here on when and how this value should be updated.